Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Solver.cs

```diff
@@ -18,24 +19,23 @@ public abstract class Solver : MonoBehaviour
         private bool updateLinkedTransform = false;
 
         [SerializeField]
-        [Tooltip("Position lerp multiplier")]
+        [Tooltip("If 0, the position will update immediately.  Otherwise, the greater this attribute the slower the position updates")]
         private float moveLerpTime = 0.1f;
 
         [SerializeField]
-        [Tooltip("Rotation lerp multiplier")]
+        [Tooltip("If 0, the rotation will update immediately.  Otherwise, the greater this attribute the slower the rotation updates")]
         private float rotateLerpTime = 0.1f;
 
         [SerializeField]
-        [Tooltip("Scale lerp multiplier")]
+        [Tooltip("If 0, the scale will update immediately.  Otherwise, the greater this attribute the slower the scale updates")]
         private float scaleLerpTime = 0;
 
         [SerializeField]
         [Tooltip("If true, the Solver will respect the object's original scale values")]
         private bool maintainScale = true;
-
         [SerializeField]
-        [Tooltip("Working output is smoothed if true. Otherwise, snapped")]
-        private bool smoothing = true;
+        [Tooltip("If true, updates are smoothed to the target. Otherwise, they are snapped to the target")]
```

Comment:
```suggestion
        [Tooltip("If true, updates are smoothed to the target. Otherwise, they are snapped to the target")]
        private bool smoothing = true;
        public bool Smoothing 
        { 
            get => smoothing; 
            set => smoothing = value; 
        }
```